src/main/java/de/maxhenkel/audioplayer/command/UtilityCommands.java

```diff
@@ -16,10 +16,39 @@
 import net.minecraft.world.InteractionHand;
 import net.minecraft.world.item.*;
 
+import java.util.ArrayList;
 import java.util.Optional;
+import java.util.UUID;
 
 @Command("audioplayer")
 public class UtilityCommands {
+    @Command("set_random")
```

Comment:
This command is missing permissions. The apply permission should be fine for this.